x/group/server/server.go

```diff
@@ -211,6 +210,6 @@ func RegisterServices(configurator servermodule.Configurator, accountKeeper expo
 	configurator.RegisterWeightedOperationsHandler(impl.WeightedOperations)
 
 	// Require servers from external modules for ADR 033 message routing
-	configurator.RequireServer((*ecocredit.MsgServer)(nil))
+	//configurator.RequireServer((*ecocredit.MsgServer)(nil))
```

Comment:
We would ideally have the group module in its current state functioning on hambach. Is this something we can easily fix rather than commenting out?

Reply:
yeah i was planning on combing back to this, but wasn't quite sure if this was even needed. i couldnt find any areas where x/group was calling the other servers.. but maybe i misunderstand how adr033 works

Reply:
Any update on this? It might not be needed.

Reply:
i think i fixed it, should just be pending https://github.com/regen-network/regen-ledger/pull/986 for sims fixes